conjureup/ui/views/applicationconfigure.py

```diff
@@ -25,45 +25,50 @@ def __init__(self, application, close_cb):
         super().__init__()
 
     def build_widget(self):
+        app.loop.create_task(self._build_widget())
+        return []
```

Comment:
Do i need this return here still?

Reply:
Yes.  `build_widget` is expected to return either a widget or a list that will be put into a `Pile`.  If you don't have a return, `None` is implied and it leads to the `NoneType has no attribute rows` error.